static/app/components/alert.tsx

```diff
@@ -78,30 +78,28 @@ function Alert({
         </IconWrapper>
       )}
       <ContentWrapper>
-        <MessageContainer
-          onClick={handleClick}
-          showIcon={showIcon}
-          showTrailingItems={showTrailingItems}
-          {...messageHoverProps}
-        >
-          <Message>{children}</Message>
-          {(showExpand || showTrailingItems) && (
-            <TrailingItemsWrap>
-              <TrailingItems onClick={e => e.stopPropagation()}>
-                {trailingItems}
-              </TrailingItems>
-              {showExpand && (
-                <ExpandIconWrap>
-                  <IconChevron direction={isExpanded ? 'up' : 'down'} />
-                </ExpandIconWrap>
-              )}
-            </TrailingItemsWrap>
+        <ContentWrapperInner>
+          <MessageContainer onClick={handleClick} {...messageHoverProps}>
```

Comment:
Btw the expandable alert's clickable area and the hover area is not the whole alert anymore but just the strip containing the message and trailing items.

Reply:
Thanks, lemme fix that